resources/schema.json

```diff
@@ -90,7 +90,33 @@
         },
         "mutators": {
             "type": "object",
-            "description": "Contains the settings for different mutations and profiles"
+            "description": "Contains the settings for different mutations and profiles",
```

Comment:
I like how php-cs-fixer does their settings validation. There every fixer class has some kind of a settings generation method that defines what settings there are for that specific fixer. 

Though that may be too much for now

Reply:
With json schema validation, we get the feedback very fast, even without running initial test suite and/or generating mutations.

We can improve (or add an additional layer of validation) later if that will be needed, but I'm not sure there is a clear reason for doing it now.